code_gen/api/khr_material_pbr.json

```diff
@@ -0,0 +1,139 @@
+                    "types" : ["ANARI_FLOAT32"],
+                    "tags" : [],
+                    "default" : 0.0,
+                    "description" : "cutoff for alpha mask mode"
+                }, {
+                    "name" : "specular",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 0.0,
+                    "description" : "specular factor"
+                }, {
+                    "name" : "specularColor",
+                    "types" : ["ANARI_FLOAT32_VEC3", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : [1.0, 1.0, 1.0],
+                    "description" : "specular color"
+                }, {
+                    "name" : "clearcoat",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 0.0,
+                    "description" : "clearcoat factor"
+                }, {
+                    "name" : "clearcoatRoughness",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 0.0,
+                    "description" : "clearcoat roughness"
+                }, {
+                    "name" : "clearcoatNormal",
+                    "types" : ["ANARI_SAMPLER"],
+                    "tags" : [],
+                    "description" : "clearcoat normal map"
+                }, {
+                    "name" : "transmission",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 0.0,
+                    "description" : "transmission factor"
+                }, {
+                    "name" : "ior",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 1.5,
+                    "description" : "index of refraction"
+                }, {
+                    "name" : "thickness",
+                    "types" : ["ANARI_FLOAT32", "ANARI_SAMPLER", "ANARI_STRING"],
+                    "tags" : ["attribute"],
+                    "default" : 0.0,
+                    "description" : "wall thickness"
+                }, {
+                    "name" : "attenuationDistance",
+                    "types" : ["ANARI_FLOAT32"],
+                    "tags" : [],
+                    "description" : "attenuation distance"
+                }, {
+                    "name" : "attenuationColor",
+                    "types" : ["ANARI_FLOAT32_VEC3"],
+                    "tags" : [],
+                    "default" : [1.0, 1.0, 1.0],
+                    "description" : "attenuation color"
+                }, {
+                    "name" : "tangent",
```

Comment:
I think the tangent is implicit (same as `normal`, which does not allow `STRING` to select an attribute either).

Reply:
I removed this and made some other additions/changes to match the ANARI-Docs PR more closely.